README.md

```diff
@@ -41,3 +41,38 @@ Almost all of the implementation was shamelessly adapted from the
 ## License
 
 [0-clause BSD license](LICENSE-0BSD.txt).
+
+## Selecting the right feature gate
+
+For some of the stm32f3xx chips there are sub-variants that behave differently
+and need to be specified.
+
+If your code built perfectly before and throws a compile error now, don't worry.
+While there are a lot of cool new features coming your way, we want to make sure
+your old code still behaves the same and there are no wrong assumptions
+creating unexpected behaviour.
+
+As this crate is still under fundamental development, expect more sub-variants
+to pop up in the future.
+
+### Steps to select the right feature
```

Comment:
In general, I think it would be more useful for the user, to not have to look at the `Cargo.toml` of this crate to find out what features are available. As a user, I want to only look at the source code as the last resort. 

Listing all available features (in this case all the sub-variants of the stm32f3xx family) explicitly in the README and maybe also in `lib.rs` would carry the information more efficiently towards the user.

Reply:
resolved in https://github.com/stm32-rs/stm32f3xx-hal/pull/75/commits/ced7a7ece4b4319ed3e03519fdf7ea19736cf5ed